compiler/rustc_typeck/src/expr_use_visitor.rs

```diff
@@ -536,55 +537,91 @@ impl<'a, 'tcx> ExprUseVisitor<'a, 'tcx> {
                         }
                         ty::BindByValue(..) => {
                             let mode = copy_or_move(mc, place);
+
+                            // Pointing to patterns in diagnostics can cause confusion,
```

Comment:
This is a hack right now, will be fixed by https://github.com/rust-lang/project-rfc-2229/issues/20